pkgs/applications/radio/sdrangel/default.nix

```diff
@@ -0,0 +1,74 @@
+{
+boost,
+cm256cc,
+cmake,
+codec2,
+fetchFromGitHub,
+fftwFloat,
+glew,
+libav,
+libiio,
+libopus,
+libpulseaudio,
+libusb,
+limesuite,
+mkDerivation,
+ocl-icd,
+opencv3,
+pkgconfig,
+qtbase,
+qtmultimedia,
+qtwebsockets,
+serialdv,
+stdenv,
```

Comment:
The comma in the end is too much (I am surprised it evaluates properly). Since `stdenv` is not used, please replace it with `lib` (and also further down then at the `meta = with lib;`)

Reply:
Oh i applied a :sort and didn't think about that ^
ok done ^

Reply:
Well i won't make those mistakes in my future packages ^